executors/sshexec/sshexec.go

```diff
@@ -26,6 +26,8 @@ import (
 	"github.com/lpabon/godbc"
 )
 
+var Sudovar string
```

Comment:
Sudovar should not be a global, it should be part of the SshExecutor object in line 31